### PR TITLE
add readme file

### PR DESCRIPTION
**Pull request by HaniffZaid**

### CODE REVIEW

**Review by HaniffZaid — COMMENTED** [review]
there's still some missing info. need some more time to finalize.

File: README.md [hunk 1]
```diff
@@ -5,36 +5,334 @@
...
+2. Nasharuddin Ghazali - [Github](https://github.com/Nash887)
+
+3. Haniff Zaid - [LinkedIn](https://www.linkedin.com/in/HaniffZaid) - [Github](https://github.com/HaniffZaid) - [Gmail](mailto:anipzaid@gmail.com)
+
+   - Project link: [Github](https://github.com/kameerahariru/KoNoHa-Property.git)
+   - Project slide presentation: [Google Slides](https://docs.google.com/presentation/d/1ll2SgYuS_S0OPC4TqHXadhiGL1vdAtEBrYCxVSpnGMA/edit?usp=sharing)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
```
Comment (kameerahariru) [inline README.md:330]: can you generate a few more paragraph about (1) instructions to the developer whom is not a coder, (2) message to contributors whom find a bug in the project and wanted to improve the code. (3) Also for a known issue in our project. (4) Ask for a donation to better improve this project. We have to make our project known. btw, very visually appealing readme.md

File: README.md [hunk 2]
```diff
@@ -5,36 +5,334 @@
...
+### 6. Clean Data
+
+We started data cleaning by:
+  1. Display summary statistics
+  2. Check for missing values
+  3. Based on the statistics, we decided to drop the Car Parks column because it comprises of 31% of overall data.
+  4. Then we trasform the data by columns
+     - Location: Strip the Kuala Lumpur words
+     - Price: Extract only numerical value
+     - Rooms: Split the value into Rooms and Store
```
Comment (Nash887) [inline README.md:248]: what is the different between rooms and store?

File: README.md (hunk 1 above)
Comment (HaniffZaid) [reply README.md:330]: ok

**Pull request closed without merging**